bin/build_atmi.sh

```diff
@@ -76,7 +76,8 @@ if [ "$1" != "nocmake" ] && [ "$1" != "install" ] ; then
    BUILDTYPE="Release"
    echo rm -rf $BUILD_AOMP/build/atmi
    rm -rf $BUILD_AOMP/build/atmi
-   MYCMAKEOPTS="-DCMAKE_INSTALL_RPATH_USE_LINK_PATH=ON -DCMAKE_INSTALL_RPATH=$AOMP_INSTALL_DIR/lib -DCMAKE_BUILD_TYPE=$BUILDTYPE -DATMI_HSA_INTEROP=on -DATMI_WITH_AOMP=on -DCMAKE_INSTALL_PREFIX=$INSTALL_ATMI -DROCM_DIR=$ROCM_DIR -DROCM_VERSION=$ROCM_VERSION -DLLVM_DIR=$LLVM_DIR"
+
+   MYCMAKEOPTS="-DCMAKE_INSTALL_RPATH_USE_LINK_PATH=ON -DCMAKE_INSTALL_RPATH=$AOMP_INSTALL_DIR/lib:\$ORIGIN/../hsa/lib -DCMAKE_BUILD_TYPE=$BUILDTYPE -DATMI_HSA_INTEROP=on -DATMI_WITH_AOMP=on -DCMAKE_INSTALL_PREFIX=$INSTALL_ATMI -DROCM_DIR=$ROCM_DIR -DROCM_VERSION=$ROCM_VERSION -DLLVM_DIR=$LLVM_DIR"
```

Comment:
There is a variable in aomp_common_vars called AOMP_ORIGIN_RPATH  set as follows. 

#Set common rpath for build scripts
if [ $AOMP_USE_HIPVDI == 0 ] ; then
   AOMP_ORIGIN_RPATH="-DCMAKE_INSTALL_RPATH_USE_LINK_PATH=ON -DCMAKE_INSTALL_RPATH=\$ORIGIN:\$ORIGIN/../hcc/lib:\$ORIGIN/../hsa/lib:$AOMP_INSTALL_DIR/lib:$AOMP_INSTALL_DIR/hcc/lib:$AOMP_INSTALL_DIR/hsa/lib"
else
   AOMP_ORIGIN_RPATH="-DCMAKE_INSTALL_RPATH_USE_LINK_PATH=ON -DCMAKE_INSTALL_RPATH=\$ORIGIN:\$ORIGIN/../hsa/lib:$AOMP_INSTALL_DIR/lib:$AOMP_INSTALL_DIR/hsa/lib"
fi